src/Relude/Extra/Newtype.hs

```diff
@@ -37,7 +37,7 @@ False
 -}
 un :: forall a n . Coercible a n => n -> a
 un = coerce
-{-# INLINE un #-}
+  {-# INLINE un #-}
```

Comment:
Looks like a redundant space was introduced somehow :slightly_smiling_face: Unfortunately, this leads to compilation error :disappointed:

Reply:
ah damn!!!